### DISCUSSION TITLE
Right Menu Expand?

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#3828] [Help] [answered]
With the latest version update, the menu that is located at the top right, expands to the entire screen, is it a bug or is it designed to behave like this?

![imagen](https://user-images.githubusercontent.com/13526704/188181812-3232caa0-661b-4606-af39-366e83b6ee4d.png)

**Comment by danharrin** [comment, marked as answer]
Run `filament:upgrade`, recompile your admin panel theme

**Reply by jcgmalave** [reply]
Thanks a lot
